docs/source/using-discovery/slurm.md

```diff
@@ -0,0 +1,769 @@
+(using-slurm)=
+# Using Slurm
+
+## Brief Overview of Slurm
+Slurm (Simple Linux Utility for Resource Management) is an open-source, highly configurable, fault-tolerant, and adaptable workload manager. It is extensively used across High-Performance Computing (HPC) environments.
+
+Slurm is designed to accommodate the complex needs of large-scale computational workloads. It can efficiently distribute and manage tasks across clusters comprising thousands of nodes, offering seamless control over resources, scheduling, and job queuing.
+It is the software on Discovery that lets users do tasks such as view information about the cluster, submit jobs on Discovery, including Job arrays, monitor jobs, view account information, check the cluster state, along with that of specific nodes.
```

Comment:
This is great. We should be sure that each of these points are covered in this .md

It might even be nice for the reader to hyperlink so that if someone is interested in how to check the status of a node they can quickly click and find it.

Reply:
I like that! Good call :)